docs/installing_deis/aws.rst

```diff
@@ -102,11 +102,24 @@ in `cloudformation.json`_. For example, to configure all of the optional setting
     {
         "ParameterKey":     "ELBScheme",
         "ParameterValue":   "internal"
+    },
+    {
+        "ParameterKey":     "RootVolumeSize",
+        "ParameterValue":   "100"
```

Comment:
They may also be confused as to what "internal" means - these options all have detailed descriptions in deis.template.json, so I'd rather not repeat those here. I tried to call them to the other file with "The default values are defined in deis.template.json."